CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 ## C library
+- Changed IA bias normalization to be consistent with A_IA=1 (#630).
+- Implemented generalized models for tracers (#630)
```

Comment:
This is a python change too, right?

Reply:
Yes, and the IAs actually happens in python. Just fixed this.